pyega3/data_file.py

```diff
@@ -138,8 +138,8 @@ def download_file(self, output_file, num_connections=1, key=None):
             DataFile.print_local_file_info('Saved to : ', output_file, check_sum)
             if not_valid_server_md5:
                 logging.info(
-                    f"WARNING: Unable to obtain valid MD5 from the server(recived:{check_sum})."
-                    f" Can't validate download. Contact EGA helpdesk")
+                    f"WARNING: Unable to obtain valid MD5 from the server (received: {check_sum})."
+                    f" Can't validate download. Please contact EGA helpdesk on ega-helpdesk@ebi.ac.uk")
```

Comment:
I added the email address so that the user doesn't have to google.

Reply:
```suggestion
                    f" Can't validate download. Please contact EGA helpdesk at helpdesk@ega-archive.org")
```

Reply:
I added the wrong email address. ALL the email addresses should be changed to this: `helpdesk@ega-archive.org`

TODO bjuhasz, afoix.